conformance/02_push_test.go

```diff
@@ -233,8 +233,8 @@ var test02Push = func() {
 				resp, err := client.Do(req)
 				Expect(err).To(BeNil())
 				location := resp.Header().Get("Location")
```

Comment:
nit: why not colocate this line with the Expect line itself? interrupts readability.

Reply:
Because this is the way it's done in every other test and this PR is about changing exactly one property of a couple of tests, not changing things which are technically fine the way they are now and might have been done for good reason.

If we were to change other aspects of these tests, there are way more important things to do, in my view (see #416 for an example).